.github/workflows/periodic-integration-tests.yml

```diff
@@ -0,0 +1,90 @@
+name: Periodic Integration Tests
+
+on:
+  schedule:
+     - cron: '0 15 * * 1-5' # Run at 7AM PST on every weekday
+  workflow_dispatch:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        # These are all the Python versions that we support.
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        concurrency: [1, 2, 5]
```

Comment:
I'm curious what insight we gain with concurrency=2 that we don't get with n=1 or n=5.

Reply:
I was thinking this would give us more clarity if we have scaling issues (ie can't handle a certain number of connections). You're right that if we have race conditions the difference between 2 and 5 won't matter, both should fail inconsistently.